dev/com.ibm.ws.persistence.ol_fat/fat/src/com/ibm/ws/persistence/fat/FATSuite.java

```diff
@@ -0,0 +1,25 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.ibm.ws.persistence.fat;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+import componenttest.custom.junit.runner.AlwaysPassesTest;
+
+@RunWith(Suite.class)
+@SuiteClasses({
+                ConsumerTest_JPA21.class,
+                //                ConsumerTest_JPA22.class,
+                AlwaysPassesTest.class })
```

Comment:
alwayspasses test is not necessary, since this FAT will run tests on the lowest java level (java 7)

Reply:
Removed